source/features/action-used-by-link.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import * as pageDetect from '../libs/page-detect';
+import searchIcon from 'octicon/search.svg';
+
+function init(): void {
+	const actionRepo = (select('.octicon-issue-opened.text-gray-dark')!.parentElement as HTMLAnchorElement)
+		.pathname.slice(1)
+		.replace('/issues', '');
```

Comment:
```suggestion
```
This was my bad i selected too many lines

Reply:
no problem